src/libcore/ops/drop.rs

```diff
@@ -21,12 +21,9 @@
 /// resource by freeing the memory or closing the file or socket. This is
 /// the job of a destructor, and therefore the job of `Drop::drop`.
 ///
-/// ## Using the `Drop` trait
+/// ## Examples
 ///
-/// While `Drop` can mostly be implemented like any other trait, there are some
-/// restrictions on how it can be used. You will see more about this later.
-///
-/// Let's first look at the following example program:
+/// Let's first take a look at the following program:
```

Comment:
```suggestion
/// To see destructors in action, let's take a look at the following program:
```
There's no "second" here, is there?

Reply:
I don't want to waste one second here :+1: